src/fsharp/FSharp.Core/seqcore.fs

```diff
@@ -403,4 +402,4 @@ namespace Microsoft.FSharp.Core.CompilerServices
             //[<System.Diagnostics.DebuggerNonUserCode; System.Diagnostics.DebuggerStepThroughAttribute>]
             member x.MoveNext() = x.MoveNextImpl()
 
-            member x.Reset() = raise <| new System.NotSupportedException()
+            member __.Reset() = raise <| new System.NotSupportedException()
```

Comment:
maybe we can use parens here, but I don't know if the `raise <|` style is prevalent in the codebase.

Reply:
raise <| happens and is fine.